File: python/day.py
from employee import Employee

class Day:
    def __init__(self, name):
        self.name = name
        self.shifts = {"morning": [], "afternoon": [], "evening": []}
    
    def assign_employee(self, shift, employee: Employee):
        if shift in self.shifts:
            self.shifts[shift].append(employee.name)
    
    def get_assigned_employees(self):
        return self.shifts

File: java/day.java
import java.util.*;

class Day {
    String name;
    Map<String, List<String>> shifts;

    public Day(String name) {
        this.name = name;
        this.shifts = new HashMap<>();
        shifts.put("morning", new ArrayList<>());
        shifts.put("afternoon", new ArrayList<>());
        shifts.put("evening", new ArrayList<>());
    }

    public void assignEmployee(String shift, Employee employee) {
        shifts.get(shift).add(employee.name);
    }

    public Map<String, List<String>> getAssignedEmployees() {
        return shifts;
    }
}